admin-dev/themes/new-theme/scss/components/_showcase-card.scss

```diff
@@ -50,6 +50,12 @@
       text-align: left;
     }
 
+     > a{
+      &.btn-outline-secondary{
+        margin-right: 15px;
+      }
+    }
```

Comment:
Please pay attention to indentation and code formatting. For example, there should be a space between selectors and opening braces, and the first line is indented with 5 spaces instead of 4.

See our [Coding Standards](https://devdocs.prestashop.com/1.7/development/coding-standards/#html-css-sass-twig-smarty-code-conventions)

In addition, consider using `rem` instead of `px`.

Reply:
I changed the indentation and add rem instead of px.